src/appComponents/tracks/TracksController.js

```diff
@@ -17,7 +17,7 @@ tracksModule.controller('TracksController',
         function($mdDialog, $sessionStorage, $rootScope, ApiJsonFactory) {
 		var tc = this;
         if ( $sessionStorage.tracks === null ||
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [There should be no spaces inside this paren.](https://www.codacy.com/app/triplez/open-event-webapp/file/2417978711/issues/source?bid=3233857&fileBranchId=3237740#l19)